internal/weapons/sword/swordofdescension/swordofdescension.go

```diff
@@ -0,0 +1,86 @@
+package swordofdescension
+
+import (
+	"fmt"
+	"github.com/genshinsim/gcsim/pkg/core"
+)
+
+func init() {
+	core.RegisterWeaponFunc("swordofdescension", weapon)
+}
+
+// Descension
+// This weapon's effect is only applied on the following platform(s):
+// "PlayStation Network"
+// Hitting enemies with Normal or Charged Attacks grants a 50% chance to deal 200% ATK as DMG in a small AoE. This effect can only occur once every 10s.
+// Additionally, if the Traveler equips the Sword of Descension, their ATK is increased by 66.
+//  * Weapon refines do not affect this weapon
+func weapon(char core.Character, c *core.Core, r int, param map[string]int) string {
+	icd := 0
+	m := make([]float64, core.EndStatType)
+
+	switch char.Key() {
+	case core.TravelerCryo,
```

Comment:
missing pyro and you can check for `key < core.TravelerDelim` instead.

Reply:
@shizukayuki Great catch TY!